include/beast/websocket/impl/read.ipp

```diff
@@ -147,23 +151,26 @@ template<class NextLayer>
 template<class DynamicBuffer, class Handler>
 void
 stream<NextLayer>::read_frame_op<DynamicBuffer, Handler>::
-operator()(error_code ec,std::size_t bytes_transferred, bool again)
+operator()(error_code ec,
+    std::size_t bytes_transferred, bool again)
 {
     using beast::detail::clamp;
+    using boost::asio::buffer;
     enum
     {
         do_start = 0,
         do_read_payload = 1,
-        do_frame_done = 3,
-        do_read_fh = 4,
-        do_control_payload = 7,
-        do_control = 8,
-        do_pong_resume = 9,
-        do_pong = 11,
-        do_close_resume = 13,
-        do_close = 15,
-        do_teardown = 16,
-        do_fail = 18,
+        do_inflate_payload = 30,
+        do_frame_done = 4,
+        do_read_fh = 5,
```

Comment:
Instead of leaving gaps in the state enum and adding, would it be easier to read and follow if you made more states? (eg `do_read_fh1` instead of `do_read_fh + 1`.)